package.json

```
@@ -28,5 +28,8 @@
     "prettier": "3.4.2",
     "typescript": "5.7.3"
   },
-  "packageManager": "pnpm@9.15.3"
+  "packageManager": "pnpm@9.15.3",
+  "engines": {
+    "node": "22.x"
```

let's:

- restrict only backwards
- allow all newer versions 
- use the latest v22 version

```suggestion
    "node": ">=22.13.0"
```